gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -540,10 +537,14 @@ def run(argv=None):
             total_base_pairs, schema_file)
         logging.info('Integer range partitioned table %s was created.',
                      table_name)
+    if not known_args.append:
+      bigquery_util.create_sample_info_table(known_args.output_table)
+    suffixes.append(bigquery_util.SAMPLE_INFO_TABLE_SUFFIX)
     load_avro = bigquery_util.LoadAvro(avro_root_path,
                                        known_args.output_table,
                                        suffixes, not known_args.append)
     not_empty_variant_suffixes = load_avro.start_loading()
+    not_empty_variant_suffixes.remove(bigquery_util.SAMPLE_INFO_TABLE_SUFFIX)
```

Comment:
You need to remove it from both this list and `suffixes`, otherwise we will end up with an extra table when `sample_lookup_optimized_output_table` is set (which should fail some of our tests where we expect 51 output tables).
Also for inputs such a gnomAD where there is no samples, `sample_info` table will be empty so this list will not have that element. I am not sure what is the `remove` behavior in that case.